projects/dendrites/permutedMNIST/analyze_results.py

```diff
@@ -41,7 +41,42 @@ def key_func(x):
     return s
 
 
-def parse_one_experiment(exp, state, df):
+def parse(best_result, results, trial_checkpoint, df_entries, exp, tag):
+    config = trial_checkpoint["config"]
+    model_args = config["model_args"]
+    kw_percent_on = model_args["kw_percent_on"]
+    weight_sparsity = model_args.get("weight_sparsity", 0.0)
+    dendrite_weight_sparsity = model_args.get("dendrite_weight_sparsity", 0.0)
+    num_segments = model_args.get("num_segments")
+    dim_context = model_args["dim_context"]
+    epochs = config["epochs"]
+    num_tasks = config["num_tasks"]
+    lr = config["optimizer_args"]["lr"]
+    momentum = config["optimizer_args"].get("momentum", 0.0)
+    iteration = results["training_iteration"]
+
+    # This list must match the column headers in collect_results
+    df_entries.append(
+        [
+            exp,
+            kw_percent_on,
```

Comment:
Here, you modify `df_entries` but don't return anything. Are you sure that it stays modified after the function call ends?